src/rgw/support/rgw_support_bundle

```diff
@@ -91,8 +91,9 @@ class RGWSupportBundle:
 
         # copy ceph crash-dump files
         if coredumps:
-            if os.path.exists(CRASHDUMP_DIR):
-                shutil.copytree(CRASHDUMP_DIR,\
+            crash_dump_dir = os.path.join(log_dir, 'rgw_debug')
+            if os.path.exists(crash_dump_dir):
+                shutil.copytree(crash_dump_dir,\
                     os.path.join(RGWSupportBundle._tmp_src, 'crash'))
```

Comment:
could you please update name "crash" -> "core_files"

Reply:
done

Reply:
why name being changed from rgw_debug to crash or core_files ? we need to retain same

Reply:
okay. I though core will be more easier which is inline with motr cores as well.
but should be okay to use 'rgw_debug' as well.

Reply:
@rohit-k-dwivedi  pls revert this change..lets use same name as 'rgw_debug'

Reply:
done @cdeshmukh @sachitanands